kie-server-parent/kie-server-controller/kie-server-controller-test-war/src/main/assembly/assembly-ee6-container.xml

```diff
@@ -50,6 +50,7 @@
 
         <include>com.sun.xml.bind:jaxb-impl</include>
         <include>com.sun.xml.bind:jaxb-core</include>
+        <include>org.slf4j:slf4j-jdk14</include>
```

Comment:
I am not 100% sure, but this could cause troubles on EAP 6 where we want the container's jboss-logmanager to be used. @sutaakar would you be able able to test this?

I know it is basically impossible to get this completely right since we use the same WAR for multiple containers...